### ISSUE TITLE
Fix all auto layout issues

### ISSUE THREAD

**Issue by nashirj** [tanjyy/ridesio#61]
Some views fit into a larger screen profile, but not in smaller ones. For example, the Create Ride Offerings Screen doesn't fit onto the iPhone 12 mini, so we might need to use a [scroll view](https://www.raywenderlich.com/5758454-uiscrollview-tutorial-getting-started). Not sure if there are other places with this same issue, but there are definitely places where the auto layout isn't set up properly, like in the Settings screen.

**Comment by tahaafzal5** [comment]
Agreed on making auto layout work on iPhone 12 Pro Max and iPhone mini only for now.
